src/context.js

```diff
@@ -27,12 +27,27 @@ export async function getContext(dir) {
 	}
 
 	if (remote) {
-		// get repo org and name from remote (URL is either HTTPS or SSH)
-		[organization, name] = remote
-			.replace(/^git@github.com:/, '')
-			.replace(/^https:\/\/github.com\//, '')
-			.replace(/.git$/, '')
-			.split('/');
+		// Removing the .git at the end
+		remote = remote.replace('.git', '');
```

Comment:
Suggested change to the replace for cases where there might be an accidental trailing slash:

`remote = remote.trim().replace(/\.git\/?$/, '');`